packages/pytest-simcore/src/pytest_simcore/helpers/faker_factories.py

```diff
@@ -58,7 +58,7 @@ def random_user(
     data = {
         # NOTE: ensures user name is unique to avoid flaky tests
         "name": f"{fake.user_name()}_{fake.uuid4()}",
-        "email": fake.email().lower(),
+        "email": f"{fake.uuid4()}_{fake.email().lower()}",
```

Comment:
I see what you are intending here but I my feeling is that you might be burring  some real concurrency bug or a faulty tear-down

Reply:
No concurrency bug here, I had a flaky test falling with duplicate key error because of the email.

Reply:
For future reference
![Screenshot 2024-09-03 at 11 22 38](https://github.com/user-attachments/assets/1a72b0c5-bf10-4b96-85fa-00a2e98f5ac0)

Reply:
... OR *faulty* teardown